src/pkg/logger/logger.go

```diff
@@ -3,20 +3,26 @@ package logger
 import (
 	"context"
 	"os"
+	"time"
 
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
 	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
 )
 
+var defaultLogLocation = "corso-" + time.Now().UTC().Format("2006-01-02T15-04-05Z") + ".log"
```

Comment:
```suggestion
var defaultLogLocation = "corso-" + common.FormatNow(common.SimpleDateTime) + ".log"
```

Reply:
`:` in `common.SimpleDateTime` is an invalid char for files on Windows. This format is similar to that but `:` replaced with `-`. I can create a new constant if you think that is necessary.